ui/app/src/utils/i18n-legacy.ts

```diff
@@ -684,12 +684,13 @@ export const publishingMessages = defineMessages({
   },
   bulkPublishNote: {
     id: 'publishing.bulkPublishNote',
-    defaultMessage: 'Bulk publish should be used to publish changes made in Studio via the UI.'
+    defaultMessage:
+      'Bulk publish should be used to publish changes made in Studio via the UI. For changes made via direct git actions, use the "Publish by..." feature.'
   },
   publishByNote: {
     id: 'publishing.publishByNote',
     defaultMessage:
-      'Publish by commit ID(s) must be used for changes made via direct git actions against the repository or pulled from a remote repository.'
+      '"Publish by..." feature must be used for changes made via direct git actions against the repository or pulled from a remote repository. For changes made via Studio on the UI, use "Bulk Publish".'
```

Comment:
Is the `...` intentional here?

Reply:
@sumerjabri yes it is, that's the label of the tab in 4.0.x

Reply:
@sumerjabri publish by commit was relabelled to "publish by..." as a consequence of the ticket that requested publishing tags.

<img width="395" alt="image" src="https://user-images.githubusercontent.com/3928341/115272995-4e15df80-a13f-11eb-9cb0-a231b04916e9.png">